src/mmw/mmw/settings/production.py

```diff
@@ -105,7 +105,6 @@
     'LayerAttribution',
     'LayerSelector',
     'LocateMeButton',
-    # 'StreamControl',
```

Comment:
There is a similar configuration still in `settings/base.py` and `core/tests.js`